frontend/mms-Admin/pages/messages.js

```diff
@@ -1,32 +1,33 @@
-import React, { useState, useEffect, useRef } from 'react'
-import ChatComponent from '../components/ChatComponent';
+import React, { useState, useEffect, useRef } from "react";
+import ChatComponent from "../components/ChatComponent";
 import styles from "../styles/messages.module.css";
 import { Avatar } from "antd";
 import Icon from "../components/Icon";
-import NotificationIcon from '../components/NotificationIcon';
-import SocketProvider from 'Context/socket';
+import NotificationIcon from "../components/NotificationIcon";
+// import SocketProvider from 'Context/socket';
 
 function Messages() {
-  
   const [selectedUser, setSelectedUser] = useState(null);
   const [currentPage, setCurrentPage] = useState(1);
   const [pageSize, setPageSize] = useState(10);
   const [usersList, setUsersList] = useState([]);
   const containerRef = useRef(null);
 
   const loadMore = () => {
-    fetch(`https://api.punkapi.com/v2/beers?page=${currentPage}&per_page=${pageSize}`)
-      .then(response => response.json())
-      .then(userItems => {
+    fetch(
+      `https://api.punkapi.com/v2/beers?page=${currentPage}&per_page=${pageSize}`,
```

Comment:
API already exist

Reply:
This screen is not part of my task. There are possible reasons why it might appear as a change I made.